### DISCUSSION TITLE
Timezone incorrect & cannot change

### DISCUSSION THREAD

**Discussion by GittyHubby2020** [tteck/Proxmox#1890] [Help Needed]
Hi,
Nextcloud pi will not load because of timezone issues.  I'm in Melbourne and our clocks changed a week ago.  NCP will not load since then.

tzselect gives the following error.
/usr/bin/tzselect: line 493: /usr/share/zoneinfo/Australia/Melbourne: Bad message
/usr/bin/tzselect: time zone files are not set up correctly

trying to change the time gives the following error.
root@nextcloudpi:~# date -s '2023-10-07 22:03:00'
date: cannot set date: Operation not permitted
Sat Oct  7 22:03:00 UTC 2023

TIA for any help.

**Comment by tteck** [comment]
Have you tried running the `dpkg-reconfigure tzdata` command?

**Comment by GittyHubby2020** [comment]
I just tried it and it didn't work.  I also ran `timedatectl list-timezones` and it only lists timezones for Africa and America.

**Reply by tteck** [reply]
Is the LXC OS up to date?

**Reply by GittyHubby2020** [reply]
I just updated it and rebooted pve.  It still has the same issue.  Also I was incorrect when I said it only listed America and Africa, I didn't realise I could scroll down.  I've been working on this issue all week and there were issues pointing towards .php issues.  I'm a noob with all this so thanks for your patience.
